src/ResourceManager/Storage/Stack/Commands.Management.Storage/help/Add-AzureRmStorageAccountNetworkRule.md

```diff
@@ -0,0 +1,205 @@
+---
+external help file: Microsoft.Azure.Commands.Management.Storage.dll-Help.xml
```

Comment:
we should remove thes efiles under the stack directory, or have the stack team fill them out